src/test-run/execute-js-expression.js

```diff
@@ -0,0 +1,19 @@
+import { createContext, runInContext } from 'vm';
+import SelectorBuilder from '../client-functions/selectors/selector-builder';
+
+export function executeSelectorExpression (expression, skipVisibilityCheck) {
+    var sandbox = {
+        Selector: (fn, options = {}) => {
+            if (skipVisibilityCheck)
+                options.visibilityCheck = false;
+
+            var builder = new SelectorBuilder(fn, options, { instantiation: 'Selector' });
```

Comment:
Here https://github.com/DevExpress/testcafe/blob/c7af41765d752e9d88beaef6e033cddb027292ca/src/test-run/commands/actions.js#L30 we catch errors during the building. Do we need to do this?